.github/workflows/deploy-to-gh-pages.yml

```diff
@@ -16,7 +16,8 @@ jobs:
 
       - name: Install and Build
         run: |
-          npm install -f 
+          npm config set legacy-peer-deps true
```

Comment:
legacy?

Reply:
Worked with Ameen for this change. It fixes the need to force install because of a conflict with required and currently used versions of React. IIRC current React version is higher than a version MUI needs, so it was erroring and needed to force install everything.